runtime/compiler/optimizer/IdiomRecognition.cpp

```diff
@@ -348,7 +348,7 @@ TR_CISCNode::printStdout()
       {
       sprintf(buf, "%d", _opcode);
       }
-   printf("[%p] %3d %2d%c %-11s", this, _id, _dagId, isOutsideOfLoop() ? ' ' : 'L', buf);
+   printf("[%p] %3d %2d%c %-11s", this, _id, _dagId, isOutsideOfLoop() ? ' ' : 'L', buf); //another unsure place for Q
```

Comment:
As above, I'm guessing this is using 'L' as short for 'loop' in debug code, so I don't think this needs to handle `Q`.